tldw_Server_API/app/api/v1/endpoints/prompt_studio_projects.py

```
@@ -221,8 +218,8 @@ async def create_project(
             if row:
                 project = db._row_to_dict(cursor, row)
                 return StandardResponse(success=True, data=ProjectResponse(**project))
-        except Exception:
-            pass
+        except Exception as fallback_err:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)